Terminal.Gui/Drawing/Color.cs

```diff
@@ -10,273 +10,275 @@
-	/// </returns>
-	/// <remarks>
-	///   If <paramref name="color" /> is equidistant from two named colors, the result of this method is not guaranteed to be determinate.
-	/// </remarks>
-	[Pure]
-	[MethodImpl (MethodImplOptions.AggressiveInlining)]
-	public static bool IsColorClosestToNamedColor (in Color color, in ColorName namedColor)
-	{
-		return color.IsClosestToNamedColor (in namedColor);
-	}
-
-	/// <summary>Gets the "closest" named color to this <see cref="Color" /> value.</summary>
-	/// <param name="inputColor"></param>
-	/// <remarks>
-	///   Distance is defined here as the Euclidean distance between each color interpreted as a <see cref="Vector3" />.
-	///   <para />
-	///   The order of the values in the passed Vector3 must be
-	/// </remarks>
-	/// <returns></returns>
-	[SkipLocalsInit]
-	internal static ColorName GetClosestNamedColor (Color inputColor) => ColorExtensions.ColorToNameMap.MinBy (pair => CalculateColorDistance (inputColor, pair.Key)).Value;
-
-	[SkipLocalsInit]
-	static float CalculateColorDistance (in Vector4 color1, in Vector4 color2) => Vector4.Distance (color1, color2);
-
-	#region Legacy Color Names
-	/// <summary>The black color.</summary>
-	public const ColorName Black = ColorName.Black;
-
-	/// <summary>The blue color.</summary>
-	public const ColorName Blue = ColorName.Blue;
-
-	/// <summary>The green color.</summary>
-	public const ColorName Green = ColorName.Green;
-
-	/// <summary>The cyan color.</summary>
-	public const ColorName Cyan = ColorName.Cyan;
-
-	/// <summary>The red color.</summary>
-	public const ColorName Red = ColorName.Red;
-
-	/// <summary>The magenta color.</summary>
-	public const ColorName Magenta = ColorName.Magenta;
-
-	/// <summary>The yellow color.</summary>
-	public const ColorName Yellow = ColorName.Yellow;
-
-	/// <summary>The gray color.</summary>
-	public const ColorName Gray = ColorName.Gray;
-
-	/// <summary>The dark gray color.</summary>
-	public const ColorName DarkGray = ColorName.DarkGray;
-
-	/// <summary>The bright bBlue color.</summary>
-	public const ColorName BrightBlue = ColorName.BrightBlue;
-
-	/// <summary>The bright green color.</summary>
-	public const ColorName BrightGreen = ColorName.BrightGreen;
-
-	/// <summary>The bright cyan color.</summary>
-	public const ColorName BrightCyan = ColorName.BrightCyan;
-
-	/// <summary>The bright red color.</summary>
-	public const ColorName BrightRed = ColorName.BrightRed;
-
-	/// <summary>The bright magenta color.</summary>
-	public const ColorName BrightMagenta = ColorName.BrightMagenta;
-
-	/// <summary>The bright yellow color.</summary>
-	public const ColorName BrightYellow = ColorName.BrightYellow;
-
-	/// <summary>The White color.</summary>
-	public const ColorName White = ColorName.White;
-	#endregion
+public readonly partial record struct Color : ISpanParsable<Color>, IUtf8SpanParsable<Color>, ISpanFormattable,
```

Comment:
Looks like the rules didn't chop this one properly.

The intent was for long lists to get chopped to one per line. Should be fairly easily fixable.